core/src/jmh/java/org/apache/iceberg/ReplaceDeleteFilesBenchmark.java

```diff
@@ -73,11 +74,14 @@ public class ReplaceDeleteFilesBenchmark {
   private List<DeleteFile> deleteFiles;
   private List<DeleteFile> pendingDeleteFiles;
 
-  @Param({"50000", "100000", "500000", "1000000", "2500000"})
+  @Param({"50000", "100000", "500000", "1000000", "2000000"})
```

Comment:
I modified the benchmark added in #11166 so that now we express a percent of data files which will have position deletes which will be replaced. I reduced this value since now we're opening up the manifest in the init of the test and with 2500000 files I get heap OOMs even after increasing the gradle heap memory to 16G. 2000000 seems to be OK and seems to be a useful enough benchmarking value? Let me know if you have any concerns @aokolnychyi .

Reply:
I believe I used `jvmArgs = ['-Xmx32g']` in `jmh.gradle`. 2M sounds fine too.